artifactory/utils/delete.go

```diff
@@ -0,0 +1,17 @@
+package utils
+
+import (
+	"fmt"
+	"github.com/jfrog/jfrog-cli-go/utils/cliutils"
+	rtclientutils "github.com/jfrog/jfrog-client-go/artifactory/services/utils"
+)
+
+func ConfirmDelete(pathsToDelete []rtclientutils.ResultItem) bool {
```

Comment:
It looks like this function has only one usage in *generic/delete.go*. I think we should move there and remove this file.

Reply:
Done